techniques/system/common/1.0/cf-served.st

```diff
@@ -60,6 +60,10 @@ bundle server access_rules
         maproot => {  @{def.acl}  },
         admit   => {  @{def.acl}  };
 
+      # Deny access to 50_techniques folder
+      "${g.rudder_ncf_origin_local}/50_techniques"
+        deny    => {  @{def.acl}  };
```

Comment:
there is no all (we could put a .* ) but I'd rather stay on the safe side , since we authorize only def.acl